libqtile/widget/generic_poll_text.py

```diff
@@ -52,8 +52,6 @@ def __init__(self, **config):
         self.add_defaults(GenPollUrl.defaults)
 
     def fetch(self, url, data=None, headers=None, is_json=True, is_xml=False):
-        if headers is None:
```

Comment:
`headers` can never be None as `fetch` is only called from `poll` and it always has `User-Agent` defined by default.

Reply:
Can you make a separate commit for this change with just this as the message?

Reply:
Of course.

Reply:
#2396